### PR TITLE
Validation for log in and sign up inputs!

### PR DESCRIPTION
**Pull request by Juf1a**

### CODE REVIEW

File: web/public/home.ejs
Comment (propelli22) [inline web/public/home.ejs:1]: move login modal to its own file, to make usage of the modal later on different pages.

File: web/public/style/main.css
Comment (propelli22) [inline web/public/style/main.css:1]: move login modal css to its own file, for usage of modal later.

**Review by propelli22 — APPROVED** [review]
OK!

Move login modal codes to their own files later.

File: web/public/home.ejs
Comment (Juf1a) [reply web/public/home.ejs:1]: Done

File: web/public/style/main.css
Comment (Juf1a) [reply web/public/style/main.css:1]: Done

**Comment by Juf1a** [comment]
Moved them all to their own files and made sure they connect to the main part

**Pull request merged**